### PR TITLE
Feature/df 1644  job center code kofax xml

### PR DESCRIPTION
**Pull request by emina-str**

### CODE REVIEW

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/messagebroker/events/DomainEventSenderGateway.java [hunk 1]
```diff
@@ -51,4 +52,9 @@ void handleProofOfWorkEffortsSubmittedEvent(ProofOfWorkEffortsSubmittedEvent pro
 				.setHeader(MessageHeaders.EVENT, proofOfWorkEffortSubmittedEvent.getDomainEventType().getValue())
 				.build());
 	}
+
+	private ApplicationDocumentEventDto toApplicationDocumentEventDto(ApplicationDocumentDto adDto, String jobCenterCode) {
```
Comment (emina-str) [inline infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/messagebroker/events/DomainEventSenderGateway.java]: For reviewer(s), please see would it be better to add just a field `jobCenterCode` to `ApplicationDocumentDto` or is it OK like so to add new class `ApplicationDocumentEventDto` only for sending Kafka messages.

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java [hunk 2]
```diff
@@ -204,6 +202,19 @@ public boolean submit(int size) {
 		return proofOfWorkEffortsResult.hasNext();
 	}
 
+	public ProofOfWorkEffortsDto findByMatchingDate(UserId userId, LocalDateTime date) {
+		ProofOfWorkEfforts proofOfWorkEfforts = findByDate(userId, date.toLocalDate());
+		return ProofOfWorkEffortsAssembler.toProofOfWorkEffortsDto(proofOfWorkEfforts);
+	}
+
+	private ProofOfWorkEfforts findByDate(UserId userId, LocalDate date) {
```
Comment (cjpsycos) [inline application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: if you throw an exception when no result is returned, then call the method getByDate(...)
same for getByMatchingDate(...)

Comment (emina-str) [reply application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: Fixed.

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/messagebroker/events/DomainEventSenderGateway.java (hunk 1 above)
Comment (emina-str) [reply infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/messagebroker/events/DomainEventSenderGateway.java]: It's alright like so.

**Pull request merged**